source/install/install-kubernetes-custom.rst

```diff
@@ -0,0 +1,73 @@
+.. _install-kubernetes-custom:
+
+Using a Custom Configuration
+=================================
+
+You can opt to install the Mattermost operator on top of your existing infrastructure in which case you only need to install the Mattermost operator:
+
+.. code-block:: sh
+
+   $ kubectl create ns mattermost-operator
+   $ kubectl apply -n mattermost-operator -f https://raw.githubusercontent.com/mattermost/mattermost-operator/master/docs/mattermost-operator/mattermost-operator.yaml
+
+Understanding Resource Requirements
+-----------------------------------
+
+Running Mattermost in Kubernetes requires different resources based on your total number of users.
+The table below details the minimum Kubernetes cluster resources that Mattermost requires at different scales.
+
+.. csv-table::
+   :header: "User Count", "Node Count", "Memory per Node", "vCPU per Node"
+   "5,000", "6", "8 GB", "4"
+   "10,000", "8", "16 GB", "4"
+   "25,000", "14", "16 GB", "4"
+
+The resource requirements, as well as the ingress name are added to the relevant field in the Mattermost manifest file, which is then deployed in your cluster.
+
+Customizing Resource Requirements
```

Comment:
This section likely needs some elaboration, especially around replication and the calculations performed. All the information is in the repo (https://github.com/mattermost/mattermost-operator/blob/master/pkg/apis/mattermost/v1alpha1/clusterinstallation_sizes.go) so we can either link to the repo, or provide the various calculations here, in which case some additional context may be needed.

Reply:
To start, I'd recommend linking to the repo - and reduce the duplication of information.  We can always improve this document with customer feedback.

Reply:
I'm going to start a new PR for this so it doesn't hold up this PR.